recipe/meta.yaml

```diff
@@ -38,28 +37,29 @@ build:
     - copy %SRC_DIR%\\lib\\x64\\cudnn*.lib %LIBRARY_LIB%\\  # [win]
     - mkdir %LIBRARY_BIN%                                   # [win]
     - copy %SRC_DIR%\\bin\\cudnn*.dll %LIBRARY_BIN%\\       # [win]
-  ignore_run_exports:
-    - cudatoolkit
+  ignore_run_exports_from:
+    - {{ compiler('cuda') }}
+    - libcublas-dev  # [(cuda_compiler_version or "").startswith("12")]
```

Comment:
My understanding is that cudnn is compatible with *any* CTK version >=11.0, so we must ignore the run_exports from libcublas-dev because they are too strict.

The real question is whether libcublas is actually used. The link checker doesn't detect any dynamic links to cublas, but the release notes mention cublas all over, so I assume it's because they are using dlopen to load cublas.

Reply:
Right I don't see any other usage of `libcublas-dev` in the recipe. We don't have `libcublas-dev` in `requirements/host`

We do have `libcublas` in `requirements/run`. Should we have `libcublas-dev` in `requirements/host`?